IPyth.sol

```diff
@@ -32,38 +32,58 @@ interface IPyth {
     /// @param fee Amount of paid fee for updating the prices.
     event UpdatePriceFeeds(address indexed sender, uint batchCount, uint fee);
 
-    /// @notice Returns the current price and confidence interval.
-    /// @dev Reverts if the current price is not available.
-    /// @param id The Pyth Price Feed ID of which to fetch the current price and confidence interval.
+    /// @notice Returns the price and confidence interval.
+    /// @dev Reverts if the price is not available.
+    /// @param id The Pyth Price Feed ID of which to fetch the price and confidence interval.
     /// @return price - please read the documentation of PythStructs.Price to understand how to use this safely.
-    function getCurrentPrice(bytes32 id) external view returns (PythStructs.Price memory price);
+    function getPrice(bytes32 id) external view returns (PythStructs.Price memory price);
 
-    /// @notice Returns the exponential moving average price and confidence interval.
-    /// @dev Reverts if the current exponential moving average price is not available.
-    /// @param id The Pyth Price Feed ID of which to fetch the current price and confidence interval.
+    /// @notice Returns the exponentially-weighted moving average price and confidence interval.
+    /// @dev Reverts if the EMA price is not available.
+    /// @param id The Pyth Price Feed ID of which to fetch the EMA price and confidence interval.
     /// @return price - please read the documentation of PythStructs.Price to understand how to use this safely.
     function getEmaPrice(bytes32 id) external view returns (PythStructs.Price memory price);
 
-    /// @notice Returns the latest available price, along with the timestamp when it was generated.
-    /// @dev This function returns the same price as `getCurrentPrice` in the case where a price was available
-    /// at the time this `PriceFeed` was published (`publish_time`). However, if a price was not available
-    /// at that time, this function returns the price from the latest time at which the price was available.
+    /// @notice Returns the price of a price feed without any sanity checks.
+    /// @dev This function returns the same price as `getPrice` in the case where the price is available.
```

Comment:
I suggest substituting "has been updated recently" for "available" here. "available" is confusing because the next sentence also says "latest available price"

Reply:
actually maybe this should say "Returns the most recent price update in this contract without any recency checks. This function is unsafe as the returned price update may be arbitrarily far in the past."